mitogen/__init__.py

```diff
@@ -35,7 +35,7 @@
 
 
 #: Library version as a tuple.
-__version__ = (0, 2, 9)
+__version__ = (0, 3, "0-rc.0")
```

Comment:
Suggestion: follow the convention of Python's `sys.version_info` and Django, keeping the first 3 elements as integers. E.g.

```python
__version__ = (0 ,3, 0, 'rc', 0)
```

Reply:
Updated! I just tested and a `pip install .` on the repo after changing `__version__` to be what you suggested resulted in the same version string as before.